examples/parcel-react/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "parcel-react",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "parcel index.html",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0"
+  },
+  "devDependencies": {
+    "@parcel/config-default": "2.13.3",
+    "parcel": "2.13.3",
+    "parcel-reporter-zephyr": "latest",
```

Comment:
When the plugin gets merged, release a latest to 0.0.35 and hard code it here.
As well as a pnpm.lock file.
We need more control over the deps, otherwise, we risk breaking them without knowledge.